doc/content/devices/models/seeed-s210x/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Seeed Studio S210x Series"
+description: ""
+weight: 
+---
+
+{{< figure src="s210x.png" alt="Generic Node Sensor Edition" class="float plain" width="80%">}}
+
+The Seeed Studio [S210x Series](https://www.seeedstudio.com/SenseCAP-S2100-LoRaWAN-Data-Logger-p-5361.html) offers users’ industrial long-distance data acquisition via LoRaWAN. This series is suitable for a wide variety of different industries such as smart agriculture, smart buildings and industrial control. 
+
+<!--more-->
+
+## Sensor Setup
+
+To acquire the provisioning information and properly setup the sensor you must download the **SenseCap Mate** App from the Play Store or the App Store on your phone.
+
+After opening the app it will ask you to log in/create an account, but you can click on **Skip** in the top right if you don't wish to sign up.
+
+Then click the device you wish to connect to (either the **S210x Sensor** or the **S2100 Data Logger**) and select configuration mode: **Advanced Configuration**. 
+
+The app then explains how to enter bluetooth pairing mode. **Press and hold the button under the device for 3 seconds and release when it starts to blink**. Once done click **Device is ready for configuration**. It will then proceed to scan for the device, and it should show up in the list. Click on **your device's serial number** to proceed.
```

Comment:
```suggestion
The app then explains how to enter Bluetooth pairing mode. Press and hold the button under the device for 3 seconds and release when it starts to blink. Once done click **Device is ready for configuration**. It will then proceed to scan for the device, and it should show up in the list. Click on **your device's serial number** to proceed.
```

Reply:
General comment that can be applied to other places and not just here - please don't use **bold** to stress sentences too much. We use bold for button names for example, so it becomes confusing if you're talking about something that can literally be seen on the screen or just stressing some important process. We usually tend to minimize content and avoid over-explaining in our docs, so I think this kind of stressing is really unnecessary. It can be used sometimes where it makes sense, but this page turns out to be flooded with bold style so it looses its point.

Reply:
Understood. I just had a feeling users might gloss over it quickly, but you are right I was using them a bit much. I will look out for it next time 👍